docs/guides/web-ui/dashboards.md

```diff
@@ -35,77 +44,125 @@ Both variants include the following metrics:
+3. Click **+ Add variable**.
+4. Define and configure your variables.
 
-#### Pie Chart
+Once defined, variables can be referenced in SQL queries using the format `$your_variable_name`
 
-A pie chart represents data as slices of a circle, where each slice represents a category or value.
+---
 
-#### Table
+## Writing Queries
 
-A table displays data in rows and columns, allowing you to present tabular data.
+Logfire uses SQL to define dashboard queries.
 
-#### Values
+If you're unsure which tables or columns are available, refer to the [records schema](explore.md#records-schema) and [metrics schema](explore.md#metrics-schema).
 
-A values chart displays a single value or multiple values as a card or panel.
+### Variable Usage
 
-#### Categories
+You can reference dashboard variables in SQL queries using the `$variable` syntax:
 
-A categories chart represents data as categories or groups, allowing you to compare different groups.
+```sql
+SELECT * FROM records WHERE service_name = $service_name
+```
 
-## Tips and Tricks
+Variables can only be used in SQL queries. They cannot be used in chart titles or other non-query fields.
 
-### Enhanced Viewing with Synchronized Tooltips and Zoom
+### Resolution Variable
 
-For dashboards containing multiple time-series charts, consider enabling "Sync Tooltip and Zoom." This powerful feature provides a more cohesive viewing experience:
+All dashboards have access to a special `$resolution` variable that can be used in your queries. This value is dynamically selected based on the dashboard's time duration to ensure optimal performance and data density. You can use it for time bucketing:
 
-**Hover in Sync:** When you hover over a data point on any time-series chart, corresponding data points on all synchronized charts will be highlighted simultaneously. This allows you to easily compare values across different metrics at the same time point.
-**Zooming Together:** Zooming in or out on a single chart will automatically apply the same zoom level to all synchronized charts. This helps you maintain focus on a specific time range across all metrics, ensuring a consistent analysis.
-Activating Sync
+```sql
+SELECT
+  time_bucket($resolution, start_timestamp) AS x,
+  count(1) as count
+FROM records
+GROUP BY x;
+```
 
-To enable synchronized tooltips and zoom for your dashboard:
+---
 
-* Open your dashboard in Logfire.
-* Click on Dashboard Setting
-* activate "Sync Tooltip and Zoom" option.
+## Layout, Duration, and Refresh
 
-### Customizing Your Charts
+Each dashboard has settings for:
 
-**Logfire** empowers you to personalize the appearance and behavior of your charts to better suit your needs.
-Here's an overview of the available options:
+* **Layout**: Panels are arranged on a grid. You can drag panels to move and resize them.
+* **Duration**: Controls the time range for the data shown (e.g., 1h, 6h, 24h).
+* **Refresh Interval**: Sets how often the dashboard automatically refreshes its data.
```

Comment:
I feel like some extra text maybe an admonition box to explain the difference between Duration and Refresh interval might be helpful here, as these are actually a bit confusing? Maybe we could add an example to illustrate the ideas? 
https://squidfunk.github.io/mkdocs-material/reference/admonitions/